cmake/external/xbyak.cmake

```diff
@@ -44,7 +44,7 @@ ExternalProject_Add(
     DEPENDS             ""
     PREFIX              ${XBYAK_PREFIX_DIR}
     SOURCE_DIR          ${XBYAK_SOURCE_DIR}
-    UPDATE_COMMAND      ""
+    # UPDATE_COMMAND      ""
```

Comment:
I commented on this line because if UPDATE_COMMAND is explicitly blank it blocks update of this project in re-run even if GIT_TAG was changed. This problem is mentioned in this cmake file: https://github.com/Shixiaowei02/Paddle/blob/da30e49f3e30151633281c048486486934ef4a7b/cmake/external/pybind11.cmake#L37

It turned out that in PR-CI-CPU-Py2, the new PR changes are also only an incremental compilation (re-build). So one solution is to delete this command.

@jczaja @luotao1 What do you think about such a solution?

Reply:
Exactly, Xbyak is going to be checked out every time.
Of course, this error was only in one CI test, because it does not build changes from scratch, which is also not clear to me why.
Of course, after adding this PR, we can remove this comment.

Reply:
@wozna xbyak is a small bunch of headers so ti shouldn't been a problem when it is checked out when rebuilding paddle. So It is good for me.